examples/history_vehicles_replacement_for_imitation_learning.py

```diff
@@ -42,7 +40,13 @@ def main(scenarios, headless, seed):
             agent = agent_spec.build_agent()
 
             smarts.switch_ego_agent({agent_id: agent_spec.interface})
-
+            smarts.history_set_start_elapsed_time(mission.start_time)
+            modified_mission = Mission(
+                goal=agent_missions[agent_id].goal,
+                start_time=.0,
```

Comment:
@Gamenot The default start_time specified in Mission class is 0.1, I tried using 0 or 0.1 here and both works. Is there a reason why the default start_time in Mission class is 0.1?

```
@dataclass(frozen=True)
class Mission:
    start: Start
    goal: Goal
    # An optional list of edge IDs between the start and end goal that we want to
    # ensure the mission includes
    route_vias: Tuple[str] = field(default_factory=tuple)
    start_time: float = 0.1
    entry_tactic: EntryTactic = None
    task: Tuple[CutIn, UTurn] = None
    via: Tuple[Via, ...] = ()

    @property
    def has_fixed_route(self):
        return not self.goal.is_endless()

    def is_complete(self, vehicle, distance_travelled):
        return self.goal.is_reached(vehicle)
```

Reply:
The reason was a thought to give a step before adding the vehicle but with that case the default should probably be a lower value like `1e-5`.